package.json

```diff
@@ -68,16 +68,16 @@
     "xpath": "~0.0.32"
   },
   "scripts": {
-    "coverage:test:core": "nyc npm run test:core",
-    "coverage:test:frontend": "nyc --nycrc-path .nycrc-frontend.json npm run test:frontend",
```

Comment:
c8 defines its own config files (e.g. .c8rc.json) but it will happily accept the nyc-named files as well. For the moment I've opted to not rename these files because it's not necessary and because I figure it makes sense to try c8 for a while before fully committing